lesson17/index.js

```diff
@@ -0,0 +1,171 @@
+const url = "https://myjson.dit.upm.es/api/bins/8b9d";
+const main = document.querySelector("main");
```

Comment:
MUST
<img width="548" alt="スクリーンショット 2022-03-04 0 30 52" src="https://user-images.githubusercontent.com/43788628/156599372-59affb0b-befc-4ee5-94b6-649d6d40cfcb.png">
Not every time, but we have found that errors occur in the sandbox. Probably because we have specified querySelector("main"); here. Fixing this in a certain way fixes it. Please look into why this is happening.